python/taulabs/uavo.py

```diff
@@ -40,7 +54,7 @@ def from_bytes(cls, data, timestamp, startOffs=0):
             unpack_field_values = cls.packstruct.unpack_from(data, offset)
 
         field_values = []
-        field_values.append(uavo.meta['name'])
+        field_values.append(cls._name)
```

Comment:
Thanks.  :)  Mostly trying to eliminate the uavo class / metadata structures-- as we start to use this as an API, they just invite people to depend upon them in fragile ways.